tests/foreman/api/test_activationkey.py

```diff
@@ -483,7 +483,7 @@ def test_positive_fetch_product_content(self):
     @upgrade
     @skip_if_not_set('fake_manifest')
     @tier2
-    @stubbed()
+    @pytest.mark.stubbed
```

Comment:
:clap: I prefer this direct marking syntax over obfuscated marks that have to be imported, and don't provide any wrapper function!